site_root/input/ImplementationGuide-NCPI-Project-Forge.json

```diff
@@ -141,6 +141,102 @@
         "reference": {
           "reference": "StructureDefinition/age-at-event"
         }
+      },
+      {
+        "description": "NCPI Project Forge StructureDefinition ncpi-document-reference, Base: DocumentReference",
+        "exampleBoolean": false,
+        "name": "NCPI Project Forge StructureDefinition/ncpi-document-reference",
+        "reference": {
+          "reference": "StructureDefinition/ncpi-document-reference"
+        }
+      },
+      {
+        "description": "NCPI Project Forge StructureDefinition drs-attachment, Base: Attachment",
+        "exampleBoolean": false,
+        "name": "NCPI Project Forge StructureDefinition/drs-attachment",
+        "reference": {
+          "reference": "StructureDefinition/drs-attachment"
+        }
+      },
+      {
+        "description": "NCPI Project Forge ResearchSubject example-subject-id",
+        "exampleBoolean": true,
+        "name": "NCPI Project Forge ResearchSubject/example-subject-id",
+        "reference": {
+          "reference": "ResearchSubject/example-subject-id"
+        }
+      },
```

Comment:
This entry should not be lingering here, right? Please re-run `fhirutil ./site_root/input/resources`.

Reply:
resolved.